pg_queries/connection.py

```
@@ -0,0 +1,21 @@
+import os
+import psycopg2
+import dotenv
+
+dotenv.load_dotenv()
+
+def Connect():
+    try:
+        conn = psycopg2.connect(
+                    dbname=os.getenv('database'),
+                        user=os.getenv('user'),
+                            password=os.getenv('password'),
+                                host=os.getenv('host')
+                                )
+        print('\nDatabase successfully connected!\n')
+
+        return conn
+
+    except:
+        print('Connection failed!')
```

I would raise an exception, unless the database is optional.
But usually is not the case.

fail-fast principle